config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "solr",
+  "index_mount": "/",
+  "config_mount": "/opt/solr/server/solr/configsets",
+  "indexes": [
+    {
+      "name": "core17",
+      "index_path": "/home/ryan/waterloo/projects/index/lucene-index.core17.pos+docvectors+rawdocs",
+      "config_path": "/home/ryan/waterloo/projects/SolrAnserini/configsets/core17"
```

Comment:
Change this to a relative path, relative to current directory, e.g., `configsets/core17`

Reply:
Added in latest commits.